gradle/libs.versions.toml

```diff
@@ -12,7 +12,7 @@ spring-boot = "3.4.3"
 swagger-core-version = "2.2.20"
 org-springframework-boot-gradle-plugin = "3.4.3"
 org-openapi-generator-gradle-plugin = "7.2.0"
-junit-jupiter = "5.12.1"
+junit-jupiter = "5.12.0"
```

Comment:
downgrade? why?

Reply:
good question, probably some git weirdness, will change it back